frontend/src/app/components/experiments/experiment-run-list/experiment-run-list.component.html

```diff
@@ -31,11 +31,17 @@ <h2>Experiment runs</h2>
         </ng-container>
 
         <ng-container matColumnDef="actions">
-            <th mat-header-cell *matHeaderCellDef> Actions </th>
+            <th mat-header-cell *matHeaderCellDef>Actions</th>
             <td mat-cell *matCellDef="let run">
-                <a mat-icon-button [routerLink]="['/experiments', 'runs', run.id]">
+                <a matTooltip="View run" mat-icon-button [routerLink]="['/experiments', 'runs', run.id]">
                     <mat-icon>visibility</mat-icon>
                 </a>
+                <!-- <button mat-icon-button matTooltip="Stop run" *ngIf="run.is_mine && !experiment.is_archived && run.state == 'IN_PROGRESS'" (click)="stopRun(run.id)">
```

Comment:
Is this still work in progress? Lets remove it if it is not going to be implemented as part of this PR.